core/assets/translators

```diff
@@ -0,0 +1,96 @@
+Prosta4okua
```

Comment:
Are you sure all of these people (I mean the whole file) are translators?

Reply:
If someone says - I'm a contributor, then he can simply remove himself from translators and add to contributors, because I don't know who is the contributor and who is the translator

Reply:
It's a very bad practice. Try looking through the pull requests and find out who is who.

Reply:
It will take a VERY long time

Reply:
Well, this is a problem on your part.

Reply:
@Voz-Duh This is only your problem, because it is you who are proposing such changes